server/db/MsSqlUtils.ts

```diff
@@ -75,10 +75,9 @@ export default class MsSqlUtils extends SqlUtils {
     return ITemptableScript;
   }
 
-  public override async createTempTable(_sql: string): Promise<string> {
+  public override createTempTable(_sql: string): ITemptableScript {
```

Comment:
Braucht es diese Funktion überhaupt? Ich glaube ich fänd's schöner wenn createTempTable nicht abstract in SqlUtils ist, sondern konkret nur in Postgres existiert und nicht in MsSql eine ganz andere Sache macht.
Glaube es ist kein guter Stil je nach DB entweder Skript auszuführen und ein Promise zurückzugeben oder einfach nur ein script zurückgeben